src/Coduo/PHPHumanizer/String/Breakpoint.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Coduo\PHPHumanizer\String;
+
+interface Breakpoint
+{
+    /**
+     * Return the length of the truncated $text depending on the $characterCount
+     *
+     * @param string $text
+     * @param int    $charactersCount
+     * @param int    $charactersCount
+     *
+     * @return int
+     */
+    public function len($text, $charactersCount);
```

Comment:
Could you please change name of this method into ``calculatePosition($text, $charactersCount)`` ?